dnf5/include/dnf5/context.hpp

```diff
@@ -52,6 +52,10 @@ class Context : public libdnf::cli::session::Session {
     /// Sets callbacks for repositories and loads them, updating metadata if necessary.
     void load_repos(bool load_system, libdnf::repo::Repo::LoadFlags flags = libdnf::repo::Repo::LoadFlags::ALL);
 
+    /// Downloads the given URLs to specified destination local paths.
+    void download_urls(
+        std::vector<std::pair<std::string, std::filesystem::path>> url2dest_path, bool fail_fast, bool resume);
```

Comment:
Nitpick, I'd prefer not to call variables/functions/arugment `x2y`, as in using `2` instead of `_to_`. But don't wanna spoil your fun if you like it :wink:

Reply:
I don't care here. I'll use _to_ as you wish.